packages/synapse-react-client/src/components/SynapseChat/SynapseChat.tsx

```diff
@@ -234,8 +265,12 @@ export function SynapseChat({
       </Alert>
     )
   }
+
   const latestTraceEventMessage =
     latestTraceEvent?.friendlyMessage ?? 'Processing...'
+
+  const showAccessLevelMenu = !sessionContext
```

Comment:
I would also make this an optional prop (default value `true`) instead of inferring it from the `sessionContext`